microsoft/PowerToys
# [Peek] Reset Sources on Peek close

## Summary of the Pull Request


## PR Checklist

- [x] **Closes:** #26160


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed
 - Preview video with Peek
 - Close Peek
 - Click on 
![image](https://github.com/microsoft/PowerToys/assets/57057282/5bee0dcc-6419-4981-9dda-0cf27b153534)
and observe that there is no notifications about Peek playing video
